### ISSUE TITLE
Allow developers to see currently staged contracts.

### ISSUE THREAD

**Issue by turbolent** [onflow/cadence#3264]
For example, maybe just link to the contract that contains all staged contracts: 
- https://testnet.flowview.app/account/0x2ceae959ed1a7e7a/storage
- https://f.dnz.dev/0x2ceae959ed1a7e7a

**Comment by turbolent** [comment]
@Aliserag could this maybe be taken care of by 4D?

**Comment by j1010001** [comment]
Low priority as a workaround exists using flowview

**Comment by turbolent** [comment]
@bluesign Awesome! 👏💪

**Comment by j1010001** [comment]
@gregsantos Can you guys publish the 2 links in the migration guide ?

**Comment by gregsantos** [comment]
@Aliserag  would this make more sense in the new doc you're working on?

**Comment by j1010001** [comment]
@Aliserag assigned to you - please figure out where the links should be published and close the issue once it is done.

**Comment by Aliserag** [comment]
We should only link to one, prefer https://f.dnz.dev/0x2ceae959ed1a7e7a due to taking fewer clicks to view the contract.

We need to minimize the number of links we're sharing. I understand the main difference between this list of staged contracts and the weekly [reports](https://github.com/onflow/cadence/tree/master/migrations_data) we're sharing is that one is updated realtime and the other gets released weekly (with a focus on errors), but the target market is the same. I would propose changing the format of the weekly reports to include a line, 'To see a realtime list of all staged contracts, click [here](https://f.dnz.dev/0x2ceae959ed1a7e7a).'  This way we only have to link to the reports and it simplifies marketing/docs etc.

In terms of discovery, I think we should add it to:
- Migration Environments [doc](https://www.notion.so/flowfoundation/Migration-Environments-Doc-dab2d0a5c7f3465e87e5f16608e55809) that is still unpublished, under how to Access Reports.
-[ Migration Guide ](https://cadence-lang.org/docs/cadence-migration-guide/#:~:text=Use%20the%20CLI%20to%20stage%20your%20contracts%20to%20Testnet%20Migration%20Environment%20for%20state%20migration%20testing.%20Automatic%20state%20migrations%20will%20happen%20several%20times%20over%20the%20Testnet%20Migration%20Environment%20testing%20period%20on%20a%20weekly%20cadence.) at the very end of the Stage and Confirm on Testnet Migration Env section: "View all staged contracts and view reports on migration status through the [migrations data repository](https://github.com/onflow/cadence/tree/master/migrations_data)."
- [Staging Guide ](https://cadence-lang.org/docs/cadence-migration-guide/staging-guide) at the end under Validate your contract.
- Lastly, I think it would be good to include on the migration hub itself under step 4 as a new link: 'Staged Contracts Report'
<img width="1353" alt="Screenshot 2024-05-07 at 4 30 54 PM" src="https://github.com/onflow/cadence/assets/11467748/f90afad3-e1e1-41f9-bfce-c7a138f81128">

**Comment by turbolent** [comment]
@Aliserag Sounds good!

**Comment by turbolent** [comment]
@gregsantos Is there anyone on DevEx who could maybe help add this?

**Comment by Aliserag** [comment]
Working on the site but made changes to the docs, @turbolent can you kindly review/approve/merge? https://github.com/onflow/cadence-lang.org/pull/100
